cts/update/array/positional-all.json

```diff
@@ -0,0 +1,64 @@
+{
+  "1-number": {
+    "request": {
+      "update": "composites",
+      "updates": [
+        {
+          "q": {
+            "_id": "array"
+          },
+          "u": {
+            "v.$[]": {
+              "$numberInt": "100"
+            }
+          }
+        }
+      ],
+      "$db": "{{.Database}}"
+    },
+    "response": {
+      "n": {
+        "$numberInt": "1"
+      },
+      "nModified": {
+        "$numberInt": "1"
+      },
+      "ok": {
+        "$numberDouble": "1.0"
+      }
+    }
+  },
+  "2-find": {
+    "request": {
+      "find": "composites",
+      "filter": {
+        "_id": "array"
+      },
+      "sort": {
+        "_id": {
+          "$numberInt": "1"
+        }
+      },
+      "$db": "{{.Database}}"
+    },
+    "response": {
+      "cursor": {
+        "firstBatch": [
+          {
+            "_id": "array",
+            "v.$[]": {
+              "$numberInt": "100"
+            }
```

Comment:
I don't quite understand this. You've inserted a document with key `v.$[]` and value 100. I think you are attempting to use positional operator right?

Reply:
Fixed